portfolio/src/main/webapp/script.js

```diff
@@ -180,6 +180,11 @@ function verifyLogin() {
         console.log(loginStatus);
         console.log(typeof(loginStatus));
         if (isLoggedIn === "true") {
+          let isAdmin = loginStatus.isAdmin;
+          if (isAdmin === "true") {
+            let deleteAllCommentsButton = document.getElementById('delete-all-comments-button');
+            deleteAllCommentsButton.innerHTML = '<button onclick="deleteAllComments()">Delete All Comments</button>';
```

Comment:
Looks good.  One improvement to think about, we should avoid having security sensitive stuff like this on the HTML/JS even if it's not displayed on the screen.  Since both HTML and JS are evaluated locally by the browser, users if they want to can view the source and learn about the button.  They can then forcefully show the button (eg. chrome dev console lets you update the html/js/css on the fly).

Alternative approach would be for the server to send down the html/js snippets for the delete button.